lib/core/src/server/build-dev.js

```diff
@@ -160,7 +160,13 @@ Would you like to run Storybook on port ${port} instead?`,
     if (program.smokeTest) {
       process.exit(stats.toJson().warnings.length ? 1 : 0);
     } else if (!program.ci) {
-      opn(address);
+      opn(address).catch(() => {
+        logger.error(`Could not open ${address} inside a browser. If you're running this command inside a
+docker container or on a CI, you need to pass the '--ci' flag to prevent opening a
+browser by default.`);
+
+        process.exit(1);
```

Comment:
Why exit here?

Let's just continue to run, opening someone's browser isn't that important.